arcade/arcade/cli/serve.py

```diff
@@ -98,13 +98,12 @@ def serve_default_actor(
 
     toolkits = Toolkit.find_all_arcade_toolkits()
     if not toolkits:
-        logger.error("No toolkits found in Python environment. Exiting...")
-        return
-    else:
-        logger.info("Serving the following toolkits:")
-        for toolkit in toolkits:
-            num_tools = sum(len(tools) for tools in toolkit.tools.values())
-            logger.info(f"  - {toolkit.name} ({toolkit.package_name}): {num_tools} tools")
+        raise RuntimeError("No toolkits found in Python environment.")
+
+    logger.info("Serving the following toolkits:")
+    for toolkit in toolkits:
+        num_tools = sum(len(tools) for tools in toolkit.tools.values())
+        logger.info(f"  - {toolkit.name} ({toolkit.package_name}): {num_tools} tools")
```

Comment:
A better fix. This ensures a non-zero exit code is returned by the default actor if no toolkits are found (consistent with the fact that this is an error condition that prevents the actor from starting).

I un-indented the following lines because the `else` condition is unnecessary.